libs/domain/search/sort/src/sort.component.ts

```diff
@@ -24,13 +23,11 @@ export class SortComponent extends LitElement {
   protected querySortValue = valueType(this.routerService.currentQuery());
 
   protected override render(): TemplateResult {
-    return this.renderSorting(this.sortingOptions);
+    return this.renderSorting();
   }
 
-  protected renderSorting(
-    sort: ProductListSort | undefined | null
-  ): TemplateResult {
-    const hasOptions = !!sort?.sortValues.length;
+  protected renderSorting(): TemplateResult {
+    const hasOptions = !!this.sortingOptions?.sortValues.length;
```

Comment:
If I correct understand `sortValues` may be undefined or array. By trying take length from `undefined` will be an error.

Reply:
The expression is checking whether sortingOptions is true and whether it has a non-zero sortValues array length.
If the sortValues array is empty or null, the expression returns 0. The current code could be sufficient